library/tinkerpop/pom.xml

```diff
@@ -48,6 +48,11 @@
             <artifactId>gremlin-core</artifactId>
             <version>${tinkerpop.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.tinkerpop</groupId>
+            <artifactId>gremlin-server</artifactId>
+            <version>${tinkerpop.version}</version>
+        </dependency>
```

Comment:
This is required because of importing the auth classes from tinkerpop?

Reply:
That's correct its for the auth classes as they are specifically for authenticating with a gremlin server